### PR TITLE
Linux update notifications

### LINKED ISSUE #257: Auto-update solution for Linux
Check for new version at same endpoint that OS X updater uses. If new version available, show notification bubble with link to GitHub Releases page. Let's be honest – they're Linux users and they'll want to read the changelog before updating ;)

### PR DESCRIPTION
**Pull request by dcposch**
Fixes #257

image

### CODE REVIEW

**Comment by dcposch** [comment]
image

File: main/auto-updater.js [hunk 1]
```diff
@@ -29,3 +31,22 @@ function init () {
...
+function checkForUpdates () {
+  // Electron's built-in auto updater only supports Mac and Windows, for now
+  if (process.platform !== 'linux') {
+    return autoUpdater.checkForUpdates()
+  }
+
+  // If we're on Linux, we have to do it ourselves
+  get.concat(config.AUTO_UPDATE_URL, function (err, res, data) {
+    if (err) return log('Error checking for app update: ' + err.message)
+    if (![200, 204].includes(res.statusCode)) return log('Error checking for app update, got HTTP ' + res.statusCode)
```
Comment (dcposch) [inline main/auto-updater.js]: @feross if things are working, the update URL always returns 200 or 204, right?

Comment (feross) [inline main/auto-updater.js]: That is correct.

File: main/auto-updater.js [hunk 2]
```diff
@@ -29,3 +31,22 @@ function init () {
...
+    if (res.statusCode !== 200) return
+
+    var obj = JSON.parse(data)
+    // TODO: version should be included in the response object, we shouldn'v have to parse obj.name
```
Comment (feross) [inline main/auto-updater.js]: I just updated the endpoint for you: https://webtorrent.io/app/update?version=0.0.0

Can you update the PR?

**Comment by feross** [comment]
Nice, lgtm! See inline comments.

File: main/auto-updater.js (hunk 2 above)
Comment (dcposch) [inline main/auto-updater.js]: word, fixed

:ship: 
:ship: 
:ship:

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "webtorrent-desktop",
   "description": "WebTorrent, the streaming torrent client. For OS X, Windows, and Linux.",
-  "version": "0.1.1",
+  "version": "0.1.0",
```
Comment (grunjol) [inline package.json]: version decrease detected

**Pull request merged**